.snippets/webservers/nginx-php8.1-nossl.conf

```diff
@@ -1,9 +1,7 @@
 server {
-    # Replace the example <domain> with your domain name or IP address
```

Comment:
This shouldn't be removed IMO. Yes, it's a bit redundant but _many_ users easily overlook `<domain>` so there is no harm in leaving it.

Reply:
Just looks weird without having the text in configs older than PHP 8.1, bu alright

Reply:
although, should be this added into SSL configs aswell as Apache? I still don't see a point in this comment

Reply:
Yes this needs to be kept as well as added to any configs that it's missing from